frontend/src/views/reading_view.js

```diff
@@ -8,24 +8,56 @@ class Segment extends React.Component {
         return (
             <div className="scroll">
                 <p>Segment Number: {this.props.segment_num + 1}</p>
-                {this.props.segmentLines.map((line, k) => (
+                {this.props.segment_lines.map((line, k) => (
                     <p key={k}>{line}</p>)
                 )}
             </div>
         )
     }
 }
 Segment.propTypes = {
-    segmentLines: PropTypes.array,
+    segment_lines: PropTypes.array,
     segment_num: PropTypes.number,
 };
 
 
+class OverviewWindow extends React.Component {
+    render() {
+        return (
+            <div className={"row"}>
+                <div className={"col-8"}>
+                    <div className="scroll_overview">
+                        {this.props.all_segments.map((el, i) => (
+                            <p key={i}>{el.text}</p>)
+                        )}
+                    </div>
+                </div>
+                <div className={"col-4"}>
+                    <p><b>Document Questions</b></p>
+                    {this.props.document_questions.map((el, i) => (
+                        <p key={i}>{el.is_overview_question ? null : el.text}</p>)
+                    )}
+                    <p><b>Overview Questions</b></p>
+                    {this.props.document_questions.map((el, i) => (
+                        <p key={i}>{el.is_overview_question ? el.text : null}</p>)
+                    )}
+                </div>
+            </div>
+        );
+    }
+}
+
+OverviewWindow.propTypes = {
+    all_segments: PropTypes.array,
+    document_questions: PropTypes.array,
+};
+
+
 class ReadingView extends React.Component {
     constructor(props){
         super(props);
         this.state = {
-            segment_num: 0,
+            segment_num: 71,
```

Comment:
Take care to remove things you've changed for prototyping convenience when you make a PR. (I usually write myself some obnoxious multiline comments to myself, so that it's really obvious when I'm scanning the diff before I open the PR that I've left something in the codebase that shouldn't be merged.)